addons/binding/org.openhab.binding.airquality/src/main/java/org/openhab/binding/airquality/internal/AirQualityConfiguration.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.airquality.internal;
+
+/**
+ * The {@link AirQualityConfiguration} is the class used to match the
+ * thing configuration.
+ *
+ * @author Kuba Wolanin - Initial contribution
+ */
+public class AirQualityConfiguration {
```

Comment:
No, the idea of the [Configuration classes](https://www.eclipse.org/smarthome/documentation/development/bindings/how-to.html#configuration) is actually to have public fields and no need for boilerplate methods. This is also done in [other bindings](https://github.com/openhab/openhab2-addons/blob/master/addons/binding/org.openhab.binding.autelis/src/main/java/org/openhab/binding/autelis/config/AutelisConfiguration.java) that way.